build-site.sh

```diff
@@ -1,16 +1,15 @@
 #!/bin/bash
 
 if [ -z "$JEKYLLSITEBUILD" ]; then
-        export JEKYLLSITEBUILD=latest
+  export JEKYLLSITEBUILD=latest
 fi
-
 if [ -z "$JEKYLL_ENV" ]; then
 	export JEKYLL_ENV=staging
 fi
 docker run \
   --cap-drop ALL \
   --rm \
-  -it \
+  -t \
   -p 4000:4000 \
   -e JEKYLL_ACTION \
   -e JEKYLL_CONFIG \
```

Comment:
@pcolmer Same comment as https://github.com/Linaro/lkft-website/pull/170: is `JEKYLL_CONFIG` still required?

Reply:
Oh 😜

Reply:
@morancj I've removed `JEKYLL_CONFIG` - thanks for catching that.